pages.ja/common/docker-ps.md

```diff
@@ -0,0 +1,36 @@
+# docker ps
+
+> Dockerコンテナ一覧を表示します。
+> 詳しくはこちら: <https://docs.docker.com/engine/reference/commandline/ps/>
```

Comment:
```suggestion
> 詳しくはこちら: <https://docs.docker.com/engine/reference/commandline/ps/>.
```

Reply:
Other Japanese translation md files (e.g., 7z.md, chsh.md, etc.) also does not use a trailing dot (.) at the end of the URL. 
Should I fix this?

Reply:
Yeah, probably. Thanks

Reply:
Sure. I'll fix it with an additional commit!

Reply:
Fixed in https://github.com/tldr-pages/tldr/pull/9785/commits/49628382d32889787e6a8d145afef8cfe5c695e7